package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "gproject",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.1",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "jest-watch-typeahead": "^0.6.5",
+    "prop-types": "^15.7.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "5.0.0",
+    "redux": "^4.1.2",
+    "web-vitals": "^2.1.2",
+    "redux-logger": "^3.0.6",
+    "react-redux": "^7.2.6",
+    "redux-thunk": "^2.4.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
```

Comment:
- [ ] It isn't possible to run the project. Kindly include the sass module package. Please fix it.

Reply:
As requested by the p[revious reviewer,  It isn't possible to run the project. Kindly include the sass module package. Please fix it.

![failure](https://user-images.githubusercontent.com/55635977/146638007-4989aa69-553c-496c-9755-8a524c650fad.png)